atom/browser/ui/certificate_trust_win.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2013 GitHub, Inc.
```

Comment:
Oh I just mean code that would be good for copy-pasting - any time you can copy-paste security code instead of writing it yourself it's a Good Thing